diff --git a/bandcamp_dl/bandcampjson.py b/bandcamp_dl/bandcampjson.py
index 5691d7c..5fb639e 100644
--- a/bandcamp_dl/bandcampjson.py
+++ b/bandcamp_dl/bandcampjson.py
@@ -1,7 +1,7 @@
 import re
 import logging
 
-import demjson
+import json
 
 
 class BandcampJSON:
@@ -48,7 +48,21 @@ class BandcampJSON:
         """Convert JavaScript dictionary to JSON"""
         logging.debug(" Converting JS to JSON..")
         self.get_js()
-        # Decode with demjson first to reformat keys and lists
-        decoded_js = demjson.decode(self.js_data)
-        # Encode to make valid JSON, add to list of JSON strings
-        self.json_data.append(demjson.encode(decoded_js))
+        s = self.js_data
+
+        # fix js objects to be valid json
+        strings = []
+        def cb(m):
+            strings.append(m.group(0))
+            return '===STRING{}==='.format(len(strings)-1)
+
+        # replace strings with placeholders
+        s = re.sub(r'"(\\.|[^"])*"', cb, s)
+        # remove comments
+        s = re.sub(r'//.*', '', s)
+        # add quotes around keys
+        s = re.sub(r'(?<=[{,\s])(\w+)(?=\s*:)', r'"\1"', s)
+        # replace strings
+        s = re.sub(r'===STRING(\d+)===', lambda m: strings[int(m.group(1))], s)
+
+        self.json_data.append(s)
diff --git a/bandcamp_dl/deps.txt b/bandcamp_dl/deps.txt
index e8fcc5d..9459760 100644
--- a/bandcamp_dl/deps.txt
+++ b/bandcamp_dl/deps.txt
@@ -1,5 +1,4 @@
 beautifulsoup4==4.6.0
-demjson==2.2.4
 docopt==0.6.2
 mutagen==1.38
 requests==2.18.4
diff --git a/requirements.txt b/requirements.txt
index 4cf6eb5..065a680 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,7 +1,6 @@
 --index-url https://pypi.python.org/simple/
 
 beautifulsoup4==4.6.0
-demjson==2.2.4
 docopt==0.6.2
 mutagen==1.38
 requests==2.18.4
diff --git a/setup.py b/setup.py
index 38c4e45..fca2cbc 100644
--- a/setup.py
+++ b/setup.py
@@ -34,7 +34,6 @@ setup(
     python_requires='~=3.4',
     install_requires=[
         'beautifulsoup4',
-        'demjson',
         'docopt',
         'mutagen',
         'requests',
